### DISCUSSION TITLE
[Feature] Add Chinese and other languages to the web page

### DISCUSSION THREAD

**Discussion by pyccl** [immich-app/immich#11628] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

In the web page, only English is displayed, and we hope to add multiple languages such as Chinese. Mobile devices have multiple languages, but they are not available on web pages.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
You can set the language in the user settings https://my.immich.app/user-settings?isOpen=app-settings
